src/material/autocomplete/autocomplete-trigger.ts

```diff
@@ -420,6 +420,20 @@ export class MatAutocompleteTrigger
 
   // Implemented as part of ControlValueAccessor.
   writeValue(value: any): void {
+    const displayWith = this.autocomplete?.displayWith ?? (value => value);
+    const valueDisplay = displayWith(value);
+
+    // Null checks are necessary here, because the autocomplete
+    // or its options may not have been assigned yet.
+    this.autocomplete?.options?.forEach(option => {
```

Comment:
I think we may have to move this into the `Promise.resolve.then` call below or we risk "Changed after checked" errors.